### PR TITLE
[2.2.x] Refs #32856 -- Clarified that psycopg2 < 2.9 is required.

### PR DESCRIPTION
**Pull request by felixxm**
See https://github.com/django/django/commit/837ffcfa681d0f65f444d881ee3d69aec23770be#commitcomment-59186889.

Follow up to 837ffcfa681d0f65f444d881ee3d69aec23770be.

### CODE REVIEW

File: docs/ref/databases.txt [hunk 1]
```diff
@@ -103,7 +103,7 @@ PostgreSQL notes
 ================
 
 Django supports PostgreSQL 9.4 and higher. `psycopg2`_ 2.5.4 through 2.8.6 is
-required, though the latest release is recommended.
+required, though the 2.8.6 is recommended.
```
Comment (kezabelle) [inline docs/ref/databases.txt:106]: `though the 2.8.6` => `though 2.8.6`?

Comment (felixxm) [reply docs/ref/databases.txt:106]: Updated.

Comment (smithdc1) [reply docs/ref/databases.txt:106]: Maybe I've not had enough coffee yet. It looks like should chop the `the` (as per Keryn's comment?)

Comment (felixxm) [reply docs/ref/databases.txt:106]: Yes, sorry. `push` without `amend` :facepalm:

**Review by smithdc1 — APPROVED** [review]

**Review by hramezani — APPROVED** [review]

**Comment by felixxm** [comment]
Thanks for reviews :+1:

**Pull request merged**